book/src/developers/designs/replication_coord.md

```diff
@@ -0,0 +1,277 @@
+                          │        │         │        │    │                         │
+                      │   └────────┘         └────────┘                              │
+                               ▲                  ▲        │                         │
+                      └ ─ ─ ─ ─│─ ─ ─ ─ ─ ─ ─ ─ ─ ┼ ─ ─ ─ ─                          │
+                               │                  │                                  │
+                               │                  │                                  │
+                               │                  │                                  │
+                            ┌──┘                  │                                  │
+                            │                     │                                  │
+                            │                     │                                  │
+┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┼ ─ ─ ─ ─             │      ┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┼ ─ ─ ─ ─
+  replicas_bne              │        │            │        replicas_syd              │        │
+│                           │                     │      │                           │
+    ┌────────┐         ┌────────┐    │            │          ┌────────┐         ┌────────┐    │
+│   │        │         │        │                 │      │   │        │         │        │
+    │   B1   │◀───────▶│   B2   │    │            └──────────│   S1   │◀───────▶│   S2   │    │
+│   │        │         │        │                        │   │        │         │        │
+    └────────┘         └────────┘    │                       └────────┘         └────────┘    │
+│                           ▲                            │        ▲                  ▲
+                            │        │                            │                  │        │
+│                           │                            │        │                  │
+                            ▼        │                            ▼                  ▼        │
+│   ┌────────┐         ┌────────┐                        │   ┌────────┐         ┌────────┐
+    │        │         │        │    │                       │        │         │        │    │
+│   │   B3   │◀───────▶│   B4   │                        │   │   S3   │◀───────▶│   S4   │
+    │        │         │        │    │                       │        │         │        │    │
+│   └────────┘         └────────┘                        │   └────────┘         └────────┘
+                                     │                                                        │
+└ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─                    └ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─
+```
+
+!!! TBD - How to remove / decomission nodes?
+
+I think origin nodes are persistent and must be manually defined. Will this require configuration of
+their server uuid in the config?
+
+Auto-node groups need to check in with periodic elements, and missed checkins.
+
+Checkins need to send ruv?
```

Comment:
I'd say so, to be able to keep track of who's up to what, and find stale nodes.

Reply:
Yeah, that was my thinking too.